metabase/metabase
# Fix legend interactions point to wrong series

Closes https://github.com/metabase/metabase/issues/40177

### Description

Fix legend interactions point to wrong series.

### How to verify

1. Create a stacked chart with a breakout
2. Hover legend items for every series, ensure the correct series is being highlighted

### Demo

Before
<img width="479" alt="Screenshot 2024-03-21 at 5 15 15 PM" src="https://github.com/metabase/metabase/assets/14301985/743600f9-2e8e-48cc-ab56-83ff40740194">

After
<img width="453" alt="Screenshot 2024-03-21 at 5 14 28 PM" src="https://github.com/metabase/metabase/assets/14301985/050f0e5f-9b1d-476b-ae6a-3e81bfd2d8c9">

### Checklist
